dev/io.openliberty.transport.http_fat/fat/src/io/openliberty/transport/http_fat/MaxOpenConnectionsTests.java

```diff
@@ -0,0 +1,483 @@
+    private static boolean runningNetty = false;
+
+    @Server("MaxOpenConnections")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        // Start the server and use the class name so we can find logs easily.
+        server.startServer(MaxOpenConnectionsTests.class.getSimpleName() + ".log");
+
+        // Go through logs and check if Netty is being used.
+        // Wait for the TCP Channel to finish loading and get the TCP Channel started message.
+        // CWWKO0219I: TCP Channel defaultHttpEndpoint has been started and is now listening for requests on host *  (IPv6) port 8010.
+        String tcpChannelMessage = server.waitForStringInLog("CWWKO0219I: TCP Channel defaultHttpEndpoint");
+        LOG.info("Endpoint: " + tcpChannelMessage);
+
+        runningNetty = tcpChannelMessage.contains(NETTY_TCP_CLASS_NAME);
+        LOG.info("Running Netty? " + runningNetty);
+
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        // Stop the server
+        if (server != null && server.isStarted()) {
+            // CWWKO0222W: TCP Channel defaultHttpEndpoint has exceeded the maximum number of open connections 2.
+            // CWWKO0029E: An exception was generated  when initializing chain CHAIN-defaultHttpEndpoint because of exception com.ibm.wsspi.channelfw.exception.ChannelException: A TCP Channel has been constructed with incorrect configuration property value, Channel Name: defaultHttpEndpoint name: maxOpenConnections value: -1 minimum Value: 1 maximum Value: 1280000
+            // CWWKO0211E: TCP Channel defaultHttpEndpoint has been constructed with an incorrect configuration property value. Name: maxOpenConnections  Value: -1  Valid Range: Minimum 1, Maximum 1280000
+            server.stopServer("CWWKO0222W", "CWWKO0029E", "CWWKO0211E");
+        }
+    }
+
+    /**
+     * Save the server configuration before each test, this should be the default server
+     * configuration.
+     *
+     * @throws Exception
+     */
+    @Before
+    public void beforeTest() throws Exception {
+        server.saveServerConfiguration();
+    }
+
+    /**
+     * Restore the server configuration to the default state after each test.
+     *
+     * @throws Exception
+     */
+    @After
+    public void afterTest() throws Exception {
+        // Restore the servers to their default state.
+        server.setMarkToEndOfLog();
+        server.setTraceMarkToEndOfDefaultTrace();
+        server.restoreServerConfiguration();
+        server.waitForConfigUpdateInLogUsingMark(null);
+    }
+
+    /**
+     * The test will set maxOpenConnections to a value of 2 and validate in the trace file that
+     * the correct value is being used.
+     *
+     * The below configuration will be used to set maxOpenConnections to 2:
+     * <tcpOptions maxOpenConnections="2"/>
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.FULL)
```

Comment:
All tests are FULL mode now so this should be a class level annotation rather than on each of the test methods. The default test was moved to another class and merged already in: https://github.com/OpenLiberty/open-liberty/pull/31388